themes/gatsby-theme-blog/src/components/bio.js

```diff
@@ -21,7 +21,7 @@ const Bio = () => {
   } = data
 
   return (
-    <Flex mb={4}>
+    <Flex css={css({ mb: 4 })}>
```

Comment:
The latest thinking here is to move away from using styled-system like style props in favor of the `css` (and `sx` prop when using the custom JSX pragma) – this helps reduce the overall bundle size of theme-ui